### PR TITLE
Starts the move of List from Typography to a component. [amends #1330]

### PR DESCRIPTION
**Pull request by humancompanion-usds**

### CODE REVIEW

File: src/_content-style-guide/bulleted-lists.md [hunk 1]
```diff
@@ -1,48 +1,37 @@
 ---
-layout: default
+layout: content-style-guide
 title: Bulleted lists
+intro-text: Bulleted lists help make information more scannable and easier to understand.  If a sentence contains more than three items or ideas, we recommend breaking them up into a bulleted list.
```
Comment (danbrady) [inline src/_content-style-guide/bulleted-lists.md:4]: We say "three or more" here, and "No single item lists" below. Are two item lists okay? Do we have to call that out?

File: src/_content-style-guide/bulleted-lists.md [hunk 2]
```diff
@@ -1,48 +1,37 @@
...
 <div class="do-dont">
-<div class="do-dont__do">
-<h3 class="do-dont__heading">Like this</h3>
-<div class="do-dont__content" markdown="1"> 
-  
- Use parallel construction and don't use ending periods for fragments.
+  <div class="do-dont__do">
+    <h3 class="do-dont__heading">Like this</h3>
+      <div class="do-dont__content" markdown="1"> 
+        Use parallel construction and don't use ending periods for fragments.
```
Comment (danbrady) [inline src/_content-style-guide/bulleted-lists.md:26]: For some reason this is getting rendered as a `pre` code block in the browser? But I don't know why.


image

File: src/_content-style-guide/bulleted-lists.md [hunk 3]
```diff
@@ -132,6 +112,4 @@ Here's what you'll need to apply:
 
 </div>
 </div>
-
-</div>
-
+</div>
```
Comment (danbrady) [inline src/_content-style-guide/bulleted-lists.md:115]: Did you want to include the component checklist here?

Comment (humancompanion-usds) [reply src/_content-style-guide/bulleted-lists.md:115]: No, component checklist is only for components, not content style guide.

File: src/_components/html/lists.html
[the remark this answers is not in the document]
Comment (humancompanion-usds) [reply src/_components/html/lists.html:1]: I added one but unfortunately Formation strips all of the formatting. So we kinda have it and we kinda don't. I really don't appreciate the overly aggressive reset in Formation, but until we get off of it we are stuck with it.

**Review by danbrady — APPROVED** [review]

**Pull request merged**